common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -136,16 +150,29 @@ export class SendTransaction extends React.Component {
                   <AmountField
                     value={value}
                     unit={unit}
+                    tokens={this.props.tokenBalances
+                      .filter(token => !token.balance.eq(0))
+                      .map(token => token.symbol)
+                      .sort()}
```

Comment:
Just a nitpick: This is some heavy lifting inside of jsx, normally I'd make this a variable in render to keep it clean.